pyprocessing/renderer/tkinter/window.py

```diff
@@ -63,3 +67,57 @@ def set_line(self, x1, y1, x2, y2):
             x1, y1, x2, y2, fill=self.stroke.hex,
         )
         self.queued_actions.append(action)
+    
+    def set_ellipse(self, x, y, width, height):
+        x1 = x - (width +1)//2
```

Comment:
spaces between operators and operands

Reply:
I didn't request a change for every single one of them, but the same comment is valid globally